src/main/java/com/project/stockexchangeappbackend/dto/StockDTO.java

```diff
@@ -20,9 +23,12 @@
     private Long id;
 
     @ApiModelProperty(notes = "The stock's name.")
+    @NotNull(message = "This field is required.")
```

Comment:
use @NotBlank, because it rejects values like " " as well (@NotNull reject only null values)